Snakefile

```diff
@@ -290,6 +290,24 @@ rule build_bus_regions:
 
 
 if config["enable"].get("build_cutout", False):
+    for technology in cutouts_in_config:
+        cutout_fl = (
+            "cutouts/" + CDIR + config["renewable"][technology]["cutout"] + ".nc"
+        )
+        if os.path.exists(cutout_fl):
+            raise Exception(
+                "A cutout file '"
+                + cutout_fl
+                + "' still exists. Stopping to avoid over-writing it"
```

Comment:
We need to improve the message to something like:
`build_cutout enabled and cutout file {c_name} risks to be overwritten. If this is an intended behaviour, please delete the file and rerun the rule, otherwise disable the build_cutout rule in the config file`

Reply:
Done! 
A cutout name wildcard doesn't work for some reason, so strings concatenation is used.